test/jasmine/tests/mapbox_test.js

```diff
@@ -475,6 +492,22 @@ describe('mapbox plots', function() {
         });
     });
 
+    it('should be able to update the access token', function(done) {
+        var promise = Plotly.relayout(gd, 'mapbox.accesstoken', 'wont-work');
+
+        promise.catch(function(err) {
+            expect(gd._fullLayout.mapbox.accesstoken).toEqual('wont-work');
+            expect(err).toEqual(new Error(constants.mapOnErrorMsg));
```

Comment:
@chriddyp @etpinard one thing we can do is to make our [token validation](https://github.com/plotly/streambed/blob/4fb60f3d5c1110ad19ba033e21d83e335850bb9d/shelly/filehst/validators.py#L56-L71) impeccable (maybe validate our token against the real mapbox api endpoint). @BRONSOLO has opened an issue about it https://github.com/plotly/streambed/issues/7010. For now in my opinion when the user supplies a bad `token/style` we catch the error then pop up an error dialogue telling them what is going on.